src/components/InventoryTable/InventoryList.test.js

```diff
@@ -50,7 +50,12 @@ describe('InventoryList', () => {
     render(
       <MemoryRouter>
         <Provider store={store}>
-          <InventoryList ref={ref} onRefreshData={onRefreshData} loaded />
+          <InventoryList
+            ref={ref}
+            onRefreshData={onRefreshData}
+            loaded
+            hasAccess={true}
```

Comment:
@bastilian It's because [I've changed](https://github.com/RedHatInsights/insights-inventory-frontend/pull/2245/files#diff-41dbf31730868a3d85a1277faa8a678ec8afb3b2a9f3605913837e182d9974beR37) default hasAccess to false, which is the correct thing to do until you fetch privileges.  It seems like a great security measure. Do you agree?